.github/workflows/notify-on-failure.yml

```
@@ -0,0 +1,24 @@
+name: Notify on Failure
```

```suggestion
name: Test Notify on Failure
```
Also could you rename the file to `test-notifiy-on-failure`?

---

Done ✅